contrib/rocketmq_proxy/filters/network/source/constant.h

```diff
@@ -0,0 +1,21 @@
+#include <string>
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace RocketmqProxy {
```

Comment:
this is a great pattern, but I have two minor suggestions:

1. you can use `constexpr absl::string_view` without incurring a fiasco, and that might be slightly faster when passing to string_view than `const char[]` because it will record the size at compile-time, rather than having search for a `'\0'`. 
2. You could also wrap a namespace `RocketmqConstants` or just `Constants`, which will make it easier to understand the source when looking at call-sites.

Reply:
I think we still need the old-style `RocketmqValues` here to avoid string initialization when they are used, this is discussed in previous PR. so @daixiang0 just rename the file and not changing the content.

Reply:
Sure, I would do this.